### PR TITLE
ci: change to using solo-charts repo and solo-containers repo

### LINKED ISSUE #647: switch to solo-charts and solo-containers
### Feature to Enhance

```markdown
Using newly created solo-charts and solo-containers to replace dependencies on full-stack-testing
```

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by JeffreyDallas**
## Description

This pull request changes the following:

* Pointing to new chart repo
* Rebrand any fst reference to solo

### Related Issues

* Closes #647

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -451,17 +451,7 @@ Once the nodes are up, you may now expose various services (using `k9s` (shift-f
...
-  kubectl port-forward svc/envoy-proxy-node1-svc -n "${SOLO_NAMESPACE}" 8181:8080 &
-  kubectl port-forward svc/envoy-proxy-node2-svc -n "${SOLO_NAMESPACE}" 8281:8080 &
-  kubectl port-forward svc/envoy-proxy-node3-svc -n "${SOLO_NAMESPACE}" 8381:8080 &
-  ```
-* Hedera explorer: `fullstack-deployment-hedera-explorer`
-  ```bash
-  #enable portforwarding for hedera explorer, can be access at http://localhost:8080/
-  kubectl port-forward svc/fullstack-deployment-hedera-explorer -n "${SOLO_NAMESPACE}" 8080:80 &
-  ```
+* Hedera explorer: `solo-deployment-hedera-explorer`
```
Comment (jeromy-cannon) [inline README.md:454]: you removed the bash examples for envoy-proxy-<node ID>-svc (which should be nodeAlias), and you removed the bash example for the hedera explorer, also.  Was this a an oversight?

File: examples/custom-network-config/Taskfile.yml [hunk 2]
```diff
@@ -5,10 +5,10 @@ dotenv:
 silent: false
 
 env:
-  FST_CHART_VERSION: v0.30.0
+  SOLO_CHART_VERSION: v0.30.0
```
Comment (jeromy-cannon) [inline examples/custom-network-config/Taskfile.yml:8]:
```suggestion
  SOLO_CHART_VERSION: v0.32.0
```

Comment (jeromy-cannon) [reply examples/custom-network-config/Taskfile.yml:8]: or should it be v0.33.0?

I noticed that there is a v0.33.0 for both charts and containers, but the release tag shows v0.32.0 and the v0.33.0 is missing from the releases.

File: version.mjs [hunk 3]
```diff
@@ -21,5 +21,5 @@
 
 export const JAVA_VERSION = '21.0.1+12'
 export const HELM_VERSION = 'v3.14.2'
-export const FST_CHART_VERSION = 'v0.30.0'
+export const SOLO_CHART_VERSION = 'v0.33.0'
```
[the remark this answers is not in the document]
Comment (JeffreyDallas) [reply version.mjs:24]: full-stack-testing repo is already on 0.32.0 a month ago,
so solo-charts is started with 0.33.0

[the remark this answers is not in the document]
Comment (jeromy-cannon) [reply version.mjs:24]: I guess I just don't understand why the release page shows 0.32.0 and is missing 0.33.0.

[the remark this answers is not in the document]
Comment (jeromy-cannon) [reply version.mjs:24]: image

[the remark this answers is not in the document]
Comment (jeromy-cannon) [reply version.mjs:24]: image

[the remark this answers is not in the document]
Comment (JeffreyDallas) [reply version.mjs:24]: I reran release workflow on solo-charts, it has 0.33.0 release now.
there seems some glitch with workflow, will need to keep an eye on it.

**Review by jeromy-cannon — APPROVED** [review]

**Pull request merged**